XavierGuertin/WildfireVisualizationProject
# #80 - feat(GeoServer): Create connection with Geoserver

### Summary
Configured GeoServer and added a test data layer for OpenLayers

### Changes Made
- Configured the GeoServer data to persist when the image is recreated via Docker
- Created a default workspace in GeoServer
- Added a data layer to GeoServer using a wildfire data shapefile
- Displayed the data using OpenLayers

### Testing Instructions
1. Run the GeoServer with Docker
2. Add dataLayer to the map
3. Reload the page to ensure that the points are displayed

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/05a01b0d-30d6-4aac-a179-ffe810404462)

### Additional Notes
- Removed MapTiler as a map provider
- The data store in GeoServer is a shapefile containing two points, but it will be the PostGIS database once it is configured. As a result, the MapView will need to be refactored to accommodate these changes.
